packages/cli/src/config/index.ts

```diff
@@ -25,6 +24,8 @@ if (inE2ETests) {
 	dotenv.config();
 }
 
+// Load schema after process.env has been overwritten
+import { schema } from './schema';
```

Comment:
I assume this is the change the PR title mentions? The other changes change from where the static dir is served from. Should that be a separate commit that is also a changelog entry?

Reply:
yeah. this was the most important change. but the removal of `InstanceSettings` from `packages/cli/src/constants.ts` is also important to make sure that `InstanceSettings` are created after `dotenv.config()` has run.

Reply:
> I assume this is the change the PR title mentions? The other changes change from where the static dir is served from. Should that be a separate commit that is also a changelog entry?

hello,How to modify the path to read env?